ast.c

```diff
@@ -1075,6 +1076,7 @@ static void ast_to_zval(zval *zv, zend_ast *ast, ast_state_info_t *state) {
 #endif
 }
 
+/* NOTE: experimental versions should always end in 0 in the future. */
```

Comment:
Why?

Reply:
Removed - that was an incorrect guess from the fact versions 50, 60, 70, and 80 didn't do that. I was unclear on the versioning scheme